### ISSUE TITLE
Write operation queue

### ISSUE THREAD

**Issue by mingard** [dadi/api#63]
**To reduce the load at times of peak read/write**

- Monitor system hardware performance (average and 98th percentile)
- When latency detected, instantly capture API write calls to a queue in memory / localdb
- Resolve queue writes at a pace that suits the load

- Possible ability to return a reference to the memory resident entry for CMS to reference
- Possibly return lock state to temporary block CMS saves
- Possibly create empty entries to create `ObjectID`'s and write field values from memory queue

**Comment by jimlambie** [comment]
Having investigated this further I don't think we need to do anything for this.

We have monitoring in place at an infrastructure level, we have a /status endpoint to obtain data about the instance, and the only time heavy writes seem to be an issue is when performing an import - in which case the import script should be written in such a way as to reduce load on API, perhaps using rate limiting.

In short, we won't proceed with development on this issue.
